src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -2492,8 +2492,8 @@ static int32_t WaitForSocketEventsInner(int32_t port, SocketEvent* buffer, int32
 }
 
 #else
-#warning epoll/kqueue not detected; building with stub socket events support
-static const size_t SocketEventBufferElementSize = sizeof(struct pollfd);
+//#warning epoll/kqueue not detected; building with stub socket events support
```

Comment:
Did you mean to leave this commented out?

Reply:
Nope. I'll remove. Thanks